### ISSUE TITLE
Feature suggestion: chat window in the Vats

### ISSUE THREAD

**Issue by edkeyes-apollo** [vessenes/persona#44]
Historically there's been a lot of value in having a parallel Discord chat going on during Vats auctions, to egg people on or to draw attention to interesting Persona features. 

So it might be worth having a simple sidebar on the site itself for people to post comments about the ongoing auctions.

**Comment by vessenes** [comment]
Yes, I've thought about that too -- Or a nudge to get into our discord maybe. Agree that a way to argue and remonstrate with others is a good plan. I think probably we should let discord manage the ins and outs of that though. I'll see if we can get a "x people are chatting about the vats" notification out of the discord API.
